packages/ocp-build/ocp-build.1.99.10-beta/opam

```diff
@@ -23,6 +23,8 @@ remove: [
   [ "rm" "-rf" "%{prefix}%/lib/ocaml/typerex/ocplib-subcmd" ]
   [ "rm" "-rf" "%{prefix}%/lib/ocaml/typerex/ocplib-system" ]
   [ "rm" "-rf" "%{prefix}%/lib/ocaml/typerex/ocplib-unix" ]
+  [ "rm" "-f" "%{prefix}%/lib/ocaml/typerex/installed.ocp" ]
+  [ "rmdir" "--ignore-fail-on-non-empty" "%{prefix}%/lib/ocaml/typerex" ]
```

Comment:
This will fail on OS X, the BSDs, and BusyBox Linux distros. It is valid for GNU `rmdir` only, as far as I know.